### DISCUSSION TITLE
Questions for fellow Csla developers

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3445] [Questions]
Hi folks,
We are a very small team. We recently got requirements for a project where we are asked to 'digitize' a few dozen forms. Although the current requirements dictate simple validation on the forms  but we know that will most likely change in the future :). 
Although there are many solutions from google forms to MS forms to ... I don't know how many others out there : )
I was wondering if fellow csla developers know of a tool (third party or other wise) that could help us develop a simple UI (for the forms) within Visual Studio (and hopefully Csla). That will save us a ton of time.
We do realize that developing a database for the 'forms' on our SQL server will take time.

Kind Regards.

**Comment by kcabral817** [comment]
Check out Radzen.com.  They have a tool that will generate the front end in blazor based on the database table.  Maybe that will help you.

**Reply by EricNgo1972** [reply]
Have a look at wisej.com. Their product allows you to design forms using drag-and-drop in Visual Studio.
And it works perfectly with csla
